litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-LIMIT-RESOURCE/test.yml

```diff
@@ -155,18 +155,43 @@
           set_fact:
             node4_id: "{{ node_id[3].split('n')[1] }}"
 
-        - name: Labeling the nodes of the cluster
-          uri:
-            url: "{{ director_url }}/v3/groups/{{ group_id }}/openebses/{{ openebsid }}/?action=selectopenebsnodes"
-            method: POST
-            url_username: "{{ username.stdout }}"
-            url_password: "{{ password.stdout }}"
-            force_basic_auth: yes
-            return_content: yes
-            body_format: json
-            body: '{"controlPlaneNodes":["{{node1_id}}"], "dataPlaneNodes":["{{node1_id}}","{{node2_id}}","{{node3_id}}","{{node4_id}}"]}'
-            status_code: 200
-          register: labelnodes
+        - block:
+          
+          - name: Get the node4_id
+            set_fact:
+              node5_id: "{{ node_id[4].split('n')[1] }}"
+          
+          - name: Labeling the nodes of the cluster
+            uri:
+              url: "{{ director_url }}/v3/groups/{{ group_id }}/openebses/{{ openebsid }}/?action=selectopenebsnodes"
+              method: POST
+              url_username: "{{ username.stdout }}"
+              url_password: "{{ password.stdout }}"
+              force_basic_auth: yes
+              return_content: yes
+              body_format: json
+              body: '{"controlPlaneNodes":["{{node1_id}}",,"{{node2_id}}","{{node3_id}}","{{node4_id}}",{{node5_id}}], "dataPlaneNodes":["{{node1_id}}","{{node2_id}}","{{node3_id}}","{{node4_id}}","{{node5_id}}"]}'
+              status_code: 200
+            register: labelnodes
+
+          when: platform == "KONVOY" or platform == "RANCHER"
+
+        - block:
+          
+          - name: Labeling the nodes of the cluster
+            uri:
+              url: "{{ director_url }}/v3/groups/{{ group_id }}/openebses/{{ openebsid }}/?action=selectopenebsnodes"
+              method: POST
+              url_username: "{{ username.stdout }}"
+              url_password: "{{ password.stdout }}"
+              force_basic_auth: yes
+              return_content: yes
+              body_format: json
+              body: '{"controlPlaneNodes":["{{node1_id}}"], "dataPlaneNodes":["{{node1_id}}","{{node2_id}}","{{node3_id}}","{{node4_id}}"]}'
```

Comment:
can we indent the body?
I think it should be possible.
```yaml
body: | # make use of this pipe symbol
  
```

Reply:
Added indented body of POST request for labeling nodes @AmitKumarDas